src/main/java/uk/gov/hmcts/reform/pip/account/management/service/AccountService.java

```diff
@@ -128,9 +134,30 @@ public Map<CreationEnum, List<?>> addUsers(List<PiUser> users, String issuerEmai
                 continue;
             }
 
+            Optional<MediaAndLegalApplication> application =
+                mediaLegalApplicationRepository.findByEmail(user.getEmail());
+
+            if (userRepository.findByEmail(user.getEmail()).isPresent()
+                && application.isPresent()) {
+                String emailSent = publicationService.sendNotificationEmailForDuplicateMediaAccount(user.getEmail(),
+                                                                            application.get().getFullName());
+                if (emailSent != null && !emailSent.isEmpty()) {
+                    ErroredPiUser erroredUser = new ErroredPiUser(user);
+                    erroredUser.setErrorMessages(Arrays.asList(emailSent));
+                    erroredAccounts.add(erroredUser);
+                }
+
+                continue;
+            }
```

Comment:
Should this not be part of the validation?

Reply:
I discussed with Chris S. This is sending email and it is not validating anything. So, keep it that way.